sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -497,4 +508,38 @@ export class ECertGenerationService {
       }
     }
   }
+
+  /**
+   * Calculate tuition remittance effective amount.
+   * @param disbursements all disbursements that are part of one e-Cert.
+   */
+  private calculateTuitionRemittanceEffectiveAmount(
```

Comment:
Please see the comment below, as a suggestion, the entire method can be reduced to the below.
```ts
private calculateTuitionRemittanceEffectiveAmount(
    disbursements: ECertDisbursementSchedule[],
  ) {
    for (const disbursement of disbursements) {
      disbursement.tuitionRemittanceEffectiveAmount =
        this.confirmationOfEnrollmentService.getMaxTuitionRemittance(
          disbursement.disbursementValues,
          disbursement.studentAssessment.application.currentAssessment.offering,
          MaxTuitionRemittanceTypes.Effective,
        );
    }
  }
```